# POTD (Pick of the Day) - James McCann

### Random Forest:

In [1]:
import pandas as pd

rf = pd.DataFrame([['Aaron Hicks', 0.379], 
                   ['Delino DeShields Jr.', 0.307], 
                   ['James McCann', 0.302], 
                   ['Eddie Rosario', 0.296], 
                   ['Victor Robles', 0.292]],
columns= ['Player', 'Probability']
)

print(rf)

                 Player  Probability
0           Aaron Hicks        0.379
1  Delino DeShields Jr.        0.307
2          James McCann        0.302
3         Eddie Rosario        0.296
4         Victor Robles        0.292


### Logistic Regression:

In [2]:
lr = pd.DataFrame([['Mookie Betts', 0.392], 
                   ['Adam Frazier', 0.329], 
                   ['James McCann', 0.321], 
                   ['Andrew Benintendi', 0.32], 
                   ['Elvis Andrus', 0.31]],
columns= ['Player', 'Probability']
)

print(lr)

              Player  Probability
0       Mookie Betts        0.392
1       Adam Frazier        0.329
2       James McCann        0.321
3  Andrew Benintendi        0.320
4       Elvis Andrus        0.310


#### James McCann

This pick may surprise some, but McCann actually has a .292 BA in 293 PA. He will see Jose Berrios, who he is 6/15 (.400) against.